tests/Feature/Controllers/Api/Journals/Modules/PrimaryObligation/PrimaryObligationControllerTest.php

```
@@ -0,0 +1,360 @@
+        $entry->refresh();
+        $this->assertEquals('none', $entry->primary_obligation);
+    }
+
+    #[Test]
+    public function it_validates_primary_obligation_must_be_valid(): void
+    {
+        $user = User::factory()->create();
+        $journal = Journal::factory()->create([
+            'user_id' => $user->id,
+        ]);
+        $entry = JournalEntry::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$entry'. (undefined)

(UnusedLocalVariable)